frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -118,6 +139,28 @@ describe('Kubevirt create VM using wizard', () => {
     );
   });
 
+  it(
+    'Creates DV with correct accessMode/volumeMode',
+    async () => {
+      const kubevirtStorageData = kubevirtStorage.data;
+      const testVMConfig = vmConfig('test-dv', provisionConfigs.get('URL'));
+      testVMConfig.networkResources = [];
+      const vm = new VirtualMachine(testVMConfig);
+
+      await withResource(leakedResources, vm.asResource(), async () => {
+        await vm.create(testVMConfig);
+        const vmDataVolume = getResourceObject(`${vm.name}-rootdisk`, vm.namespace, 'dv');
+        expect(vmDataVolume.spec.pvc.accessModes[0]).toEqual(
```

Comment:
just a thought: shouldn't we also check that there is any accessMode and volumeModa defined in our config map, so we don't check only nulls here?

Reply:
Good point